src/main/scala/io/iohk/ethereum/blockchain/sync/FastSync.scala

```diff
@@ -266,10 +276,16 @@ trait FastSync {
       requestedReceipts = requestedReceipts - handler
     }
 
+    private def validateBlocks(requestedHashes: Seq[ByteString], blockBodies: Seq[BlockBody]): Boolean = (requestedHashes zip blockBodies)
```

Comment:
I really really hate to throw away good blocks. They are hard to come by. Is there a compelling reason to?